src/utils/crypto/index.ts

```diff
@@ -200,7 +202,16 @@ export async function encryptWithRecoveryKey(key: string) {
 export default CryptoWorker;
 
 export async function getFileHash(file: File | ElectronFile) {
-    const stream = await file.stream();
-    const hash = await cryptoGenericHash(stream);
+    let filedata: DataStream;
+    if (file instanceof File) {
+        filedata = getFileStream(
+            new FileReader(),
```

Comment:
creating a new FileReader every time can cause memory issues, it would be better if we create a instance of file reader in `extractMetadataFromFiles`  and pass that for every file